open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>Figure was not successfully converted. Using ":::" instead of "<code></code>`"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>